cloud/pkg/cloudstream/streamserver.go

```diff
@@ -106,7 +109,23 @@ func (s *StreamServer) getContainerLogs(r *restful.Request, w *restful.Response)
 		}
 	}()
 
-	sessionKey := strings.Split(r.Request.Host, ":")[0]
+	// extract pod namespace and pod name from request
+	meta := strings.Split(r.Request.URL.Path, "/")
+	namespaceName := meta[2]
+	podName := meta[3]
+
+	kubeClient := client.GetKubeClient()
```

Comment:
Can use `informer` here?

Reply:
Can you describe your point a little bit more or can you add a reference to the informer, please?

Reply:
Please see `cloud/pkg/common/informers`.
It may be better for `get` or `list` to request data from the cache.

Reply:
Yes, requesting data from the cache might be better. But actually there are not the information which we need. From the `kubectl log` request we know the name of the namespace and the name of the pod. So currently I think it is easier to query the apiserver. The usage of the cache can be added in a different request. To track this we can create an issue. Is this ok for you?

Reply:
@zc2638 let's fix this bug first and create an issue for the caching later on